<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFAnnotationDlg</class>
 <widget class="QDialog" name="PDFAnnotationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbUser">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbModDate">
     <property name="text">
      <string>mod Date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbType">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
